services/aes/src/lib.rs

```diff
@@ -44,7 +44,8 @@ pub use vex::{Aes128, Aes256};
 #[cfg(feature = "vexii-test")]
 mod zkn;
 #[cfg(feature = "vexii-test")]
-pub use soft::Aes128Soft as Aes128;
+// pub use soft::Aes128Soft as Aes128;
```

Comment:
Did this accidentally get included here as part of testing? Or should this comment be removed?

Reply:
This started as a patch to optimize AES implementations and grew into a patch that optimized kernel message passing. I guess properly speaking it should be two separate PR's, but "meh".